tfx/experimental/pipeline_testing/stub_component_launcher.py

```diff
@@ -38,6 +38,56 @@ class StubComponentLauncher(
   stubbed_component_map = ...  # type: Dict[Text, Type[base_stub_executor.BaseStubExecutor]]
   test_data_dir = ...  # type: Text
 
+  @classmethod
+  def get_stub_launcher_class(
+      cls,
+      test_data_dir: Text,
+      stubbed_component_ids: List[Text],
+      stubbed_component_map: Dict[Text, \
```

Comment:
It seems that '\' is not required here.

https://google.github.io/styleguide/pyguide.html#32-line-length